kotsadm/operator/pkg/supportbundle/server.go

```diff
@@ -1,48 +0,0 @@
-package supportbundle
-
-import (
-	"fmt"
-	"net/http"
-	"runtime/pprof"
-
-	"github.com/google/martian/log"
-	"github.com/gorilla/mux"
-)
-
-func StartServer() {
-	go func() {
```

Comment:
This server is needed.  This is how support bundle gets go-routine dump from the operator service